src/js/media-chrome-range.js

```diff
@@ -213,6 +216,16 @@ class MediaChromeRange extends window.HTMLElement {
         const mediaControllerEl = document.getElementById(newValue);
         mediaControllerEl?.associateElement?.(this);
       }
+    } else if (
+      attrName === 'disabled' ||
+      attrName === 'aria-disabled' &&
+      oldValue !== newValue
+    ) {
+      if (newValue == null) {
+        this.range.removeAttribute(attrName);
+      } else {
+        this.range.setAttribute(attrName, newValue);
```

Comment:
we want to copy over the `disabled` attributes onto the `input` to make sure that it gets disabled correctly.